pkg/actuators/machine/actuator.go

```diff
@@ -254,7 +254,7 @@ func (gl *glogLogger) Logf(format string, v ...interface{}) {
 // DeleteMachine deletes an AWS instance
 func (a *Actuator) DeleteMachine(cluster *clusterv1.Cluster, machine *clusterv1.Machine) error {
 	// Drain node before deleting
-	if machine.ObjectMeta.Annotations["openshift.io/drain-node"] == "True" && machine.Status.NodeRef != nil {
+	if _, exists := machine.ObjectMeta.Annotations["openshift.io/drain-node"]; exists && machine.Status.NodeRef != nil {
```

Comment:
@ingvagabund once we finish the API pivoting, let's drain by default, and make optional opt-out
cc @derekwaynecarr

Reply:
@enxebre yes - is this enabled now?

Reply:
about to make it enabled by default, just implementing supporting integration tests